ush/create_aqm_rc_file.py

```diff
@@ -65,8 +65,8 @@ def create_aqm_rc_file(cdate, run_dir, init_concentrations):
     aqm_rc_bio_file_fp=os.path.join(AQM_BIO_DIR, AQM_BIO_FILE)
     aqm_fire_file_fn=AQM_FIRE_FILE_PREFIX+"_"+yyyymmdd+"_t"+hh+"z"+AQM_FIRE_FILE_SUFFIX
     aqm_rc_fire_file_fp=os.path.join(COMINext, "FIRE_EMISSION", aqm_fire_file_fn)
-    aqm_fengsha_file_fn=AQM_FENGSHA_FILE_PREFIX+"_"+PREDEF_GRID_NAME+AQM_FENGSHA_FILE_SUFFIX
-    aqm_rc_fengsha_file_fp=os.path.join(AQM_FENGSHA_DIR, aqm_fengsha_file_fn)
+    aqm_dust_file_fn=AQM_DUST_FILE_PREFIX+"_"+PREDEF_GRID_NAME+AQM_FENGSHA_FILE_SUFFIX
```

Comment:
AQM_FENGSHA_FILE_SUFFIX -> AQM_DUST_FILE_SUFFIX

Reply:
https://github.com/bbakernoaa/ufs-srweather-app/blob/feature/dust_aqmrc/ush/create_aqm_rc_file.py#L68-L69

Reply:
its correct in the branch

Reply:
NVM I see